packages/backend/src/accounting/service.test.ts

```diff
@@ -302,8 +302,8 @@ describe('Accounting Service', (): void => {
       `('$description', ({ sourceAmount, destinationAmount }): void => {
         test.each`
           commit   | description
-          ${true}  | ${'commit'}
-          ${false} | ${'rollback'}
+          ${true}  | ${'post'}
+          ${false} | ${'void'}
         `('$description', async ({ commit }): Promise<void> => {
```

Comment:
`commit` --> `post`

Reply:
Renaming this `commit`?
https://github.com/interledger/rafiki/blob/29211d0f66ee1d00965a47af3f27163ba3bb6114/packages/backend/src/accounting/service.test.ts#L304

If that's right, I _think_ it can also be renamed in `createTransfers`.
https://github.com/interledger/rafiki/blob/29211d0f66ee1d00965a47af3f27163ba3bb6114/packages/backend/src/accounting/transfers.ts#L29-L33

Reply:
+1 for more consistency